### PR TITLE
Issue #1471: Output feedback module for HTML deliverables only

### PR DESCRIPTION
**Pull request by Lennonka**
I modified the condition for output of the feedback module in Satellite.

Expects reversal of removed l10n markers as mentioned in https://github.com/theforeman/foreman-documentation/issues/1471#issuecomment-1189198094

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3

### CODE REVIEW

**Review by maximiliankolb — APPROVED** [review]
Looks like a very elegant solution. If that works, you should be good to re-add the l10n markers.

**Comment by Lennonka** [comment]
@maximiliankolb And should I just re-add them in this PR, so we can see that the build is passing?

**Comment by maximiliankolb** [comment]
@Lennonka Sure! Preferably as separate commit.

File: guides/doc-Administering_Project/master.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@ include::common/header.adoc[]
 
 = {AdministeringDocTitle}
 
-ifdef::satellite[]
+ifeval::["{build}" == "satellite" && "{backend}" == "html5"]
```
Comment (ekohl) [inline guides/doc-Administering_Project/master.adoc:7]: This is a lot of duplication. Can this `ifeval` be placed in a file to include? Would that be an assembly?

Comment (Lennonka) [reply guides/doc-Administering_Project/master.adoc:7]: You mean to place it inside the feedback module file?

Comment (Lennonka) [reply guides/doc-Administering_Project/master.adoc:7]: I'd prefer to keep it this simple way, it's easiest to read in the source code...

Comment (Lennonka) [reply guides/doc-Administering_Project/master.adoc:7]: I say no.

Comment (ekohl) [reply guides/doc-Administering_Project/master.adoc:7]: @maximiliankolb thoughts on this?

**Comment by Lennonka** [comment]
Yipeee, PDF is building!

File: guides/doc-Administering_Project/master.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/doc-Administering_Project/master.adoc:7]: There's a third option: place this into your file:

```
ifeval::["{build}" == "satellite" && "{backend}" == "html5"]
...
my content
...
endif::[]
```

similar to the firewalld snippet.

Comment (Lennonka) [reply guides/doc-Administering_Project/master.adoc:7]: I think it's best if the condition is visible in master.adoc for orientation in the code.

**Review by maximiliankolb — APPROVED** [review]

**Comment by Lennonka** [comment]
Cherry-picked:
- 3e88b75fc..8aebd9e34  3.4 -> 3.4
- 7c6398346..a31f5c0df  3.3 -> 3.3
- 169676bcf..d304c8194  3.2 -> 3.2
- 520846d9e..8595f306d  3.1 -> 3.1

**Comment by Lennonka** [comment]
Cherry-picked 3ecccf5e5:

- 8aebd9e34..2445cbb66  3.4 -> 3.4
- a31f5c0df..f70230e08  3.3 -> 3.3
- d304c8194..4a123298d  3.2 -> 3.2
- 8595f306d..904d13f9d  3.1 -> 3.1

**Pull request merged**